src/app/query/Cart.query.js

```diff
@@ -106,6 +106,36 @@ export class CartQuery {
         ];
     }
 
+    _getCustomizableOptionPriceFields() {
```

Comment:
`_getCustomizableOptionPriceField` and `_getCustomizableOptionPriceFields` - please split all 3 function bellow for better extensisiblity, the `_get...Fields` must return an array.